### PR TITLE
Insert child helper

### PR DESCRIPTION
**Pull request by BenKandelaars**
Function inserts a DOM element as a child and offers some helpful standardised clean up

### CODE REVIEW

File: test/dom.js [hunk 1]
```diff
@@ -270,6 +272,18 @@ describe('dom', function () {
...
+
+  describe('insertChild', function () {
+    it('should insertChild an element and return a restore function', function () {
+      const dummy = document.createElement('div')
+      dummy.id = 'dummy'
+      const restore = insertChild(two, dummy)
+      expect(fromArray(container.children)).to.eql([one, two, three])
+      expect(fromArray(two.children)).to.eql([dummy])
+      restore()
+      expect(fromArray(container.children)).to.eql([one, two, three])
```
Comment (liviarett) [inline test/dom.js:284]: should we check here if `two.children` has been removed?

File: dom/index.js [hunk 2]
```diff
@@ -110,14 +110,20 @@ function insertBefore (target, el) {
   return once(() => parent.removeChild(el))
 }
 
+function insertChild (target, el) {
```
Comment (bobertyy) [inline dom/index.js:113]: _Petty Comment_: Should we call this `appendChild` to clarify the location of the insertion?

File: test/dom.js (hunk 1 above)
Comment (BenKandelaars) [reply test/dom.js:284]: sure

File: dom/index.js (hunk 2 above)
Comment (BenKandelaars) [reply dom/index.js:113]: okay

**Review by liviarett — APPROVED** [review]

**Comment by alanclarke** [comment]
looks great @BenKandelaars ! do you mind updating the readme?

**Pull request merged**